specification/automation/resource-manager/Microsoft.Automation/stable/2022-01-31/deletedAutomationAccount.json

```diff
@@ -0,0 +1,147 @@
+{
+  "swagger": "2.0",
+  "info": {
```

Comment:
Can you please follw the checklist process described here : 


Adding a new API version
-[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits.

Reply:
As I have committed all the changes already, what can I do now?